mempool/cat/pool_test.go

```diff
@@ -747,7 +748,10 @@ func TestTxPool_ConcurrentlyAddingTx(t *testing.T) {
 			errCount++
 		}
 	}
-	require.Equal(t, numTxs-1, errCount)
+	// At least one tx should succeed and get added to the mempool without an error.
```

Comment:
>At least one tx should succeed and get added to the mempool without an error.

I've already approved it, but I was thinking we should maybe set it to 2, or something higher than 1, to ensure the concurrent adding works properly.